binding.cc

```diff
@@ -1107,6 +1197,77 @@ namespace zmq {
     BufferReference* bufref_;
   };
 
+  NAN_METHOD(Socket::Sendv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::False());
```

Comment:
Could use a primitive setter and just do `return info.GetReturnValue().Set(false);`.